addons/medical_gui/stringtable.xml

```diff
@@ -3,12 +3,15 @@
     <Package name="Medical GUI">
         <Key ID="STR_ACE_Medical_GUI_Category">
             <English>ACE Medical GUI</English>
+            <German>ACE Sanitäter Oberfläche</German>
```

Comment:
Interface maybe?

Reply:
Do you really think Interface is better then Oberfläche? :D

Reply:
A table has a "Oberfläche" too. Is that really the German term for GUI?
"Benutzeroberfläche"
Or might aswell just keep it "GUI", we just have to translate to be understandable, don't need to translate every word if germans use denglisch.

Reply:
How about arsenal stringtable L689
https://github.com/acemod/ACE3/pull/6920/files#diff-3cb47480653b94a95c0c9fd113f67c73R689
"Menü" Though that implies a context menu/dropdown/interaction menu thing.

Reply:
<German>Sanitätsmenü</German>

would use this then